packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -264,6 +280,14 @@ export class NotifiFrontendClient {
 
     let loginResult: Types.UserFragmentFragment | undefined = undefined;
     switch (walletBlockchain) {
+      case 'INJECTIVE':
```

Comment:
Not sure should INJECTIVE belong to here.
Seems INJECTIVE requires publickkey but also accountId.